include/aligator/modelling/dynamics/integrator-abstract.txx

```diff
@@ -0,0 +1,13 @@
+/// @copyright Copyright (C) 2024 LAAS-CNRS, INRIA
+#pragma once
+
+#include "./integrator-abstract.hpp"
```

Comment:
On other txx file you're doing the following :

```cpp
#include "aligator/context.hpp"
#include "aligator/core/integrator-abstract.hpp"
```
I prefer the syntax with the absolute path and the context.hpp included before.

Reply:
Alright, will fix!